src/include/kernel/x86_64/lapic.h

```diff
@@ -49,9 +49,13 @@ void init_local_vector_table();
 
 void start_apic_timer(uint16_t, bool);
 void write_apic_register(uint32_t, uint32_t);
+void write_apic_icr_register(uint64_t);
```

Comment:
i'm wrong or you added this header without implementing the function? is it intentional?